packages/core/src/components/workflow-steps/test/workflow-steps.ct.ts

```diff
@@ -81,3 +73,36 @@ regressionTest('should prevent click navigation', async ({ mount, page }) => {
   await expect(firstStepDiv).toHaveClass(/selected/);
   await expect(lastStepDiv).not.toHaveClass(/selected/);
 });
+
+regressionTest(
+  'should have the correct visuals after toggling state from open to error and back again to open',
+  async ({ mount, page }) => {
+    await mount(`
+      <ix-workflow-steps clickable vertical>
+      <ix-workflow-step id="step1" status='open'>Step 1</ix-workflow-step>
+      <ix-workflow-step status='success'>Step 2</ix-workflow-step>
+      <ix-workflow-step status='error'>Step 3</ix-workflow-step>
+      </ix-workflow-steps>
+    `);
+    const workflowSteps = page.locator('ix-workflow-steps');
+    const step1 = page.locator('#step1');
+    const selectedDiv = step1.locator('.step');
+    await expect(workflowSteps).toHaveClass(/hydrated/);
+    await expect(selectedDiv).toHaveClass(/selected/);
+    let icon = await page.locator('#step1 ix-icon').nth(1);
+    let iconAriaLabel = await icon.getAttribute('aria-label');
+    await step1.evaluate((el) => {
+      el.setAttribute('status', 'error');
+    });
+    await expect(step1).toHaveAttribute('status', 'error');
+    iconAriaLabel = await icon.getAttribute('aria-label');
```

Comment:
We can remove declaration above
```suggestion
    let iconAriaLabel = await icon.getAttribute('aria-label');
```